client/clientservice/src/event_service.cpp

```diff
@@ -47,6 +50,8 @@ Status EventServiceImpl::Subscribe(ServerContext* context,
                                    ServerWriter<SubscribeResponse>* stream) {
   LOG_INFO(logger_, "");
   cc::SubscribeRequest request;
+  static const auto historgram_log_duration_microsec{std::chrono::microseconds(300s).count()};
```

Comment:
Are these options going to have configurable values in near future?
Is TODO note required here?

Reply:
1) I don't see why we would like to configure this from the outside.
This is a hard-coded configurable value, and currently, things still don't work well (pending task).
It put it here and moved it from the middle of the code to give better visibility, so this is probably the most we should do to configure this.

2) I'm going to move them to be class members, even though we have a single subscribe by design, it is not a good thing to keep it static if by any chance we will have more than a single subscribe in the future.